src/app/components/profile-membership/profile-membership.component.html

```diff
@@ -23,7 +23,7 @@ <h6>Status:</h6>
                     </select>
                     
             
-                <button type="submit" class="btn btn-primary myBtn">Save</button>
+                <button type="submit" (click)="updatesMembershipInfo()" class="btn btn-primary myBtn">Save</button>
```

Comment:
Where does this method come from and what does it do?

Reply:
This event actually lets us call the function that updates the membership info, in the `profile-membership` controller.

Reply:
Alrighty, thank you.